### ISSUE TITLE
Pull and publish use the same worker

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#1300]
My MISP is syncing with a large remote server. Each pull cycle from this server takes more than 1 hour. During this time publishing event are queued until the pull completes. A publishing delay of 1 hour is not acceptable for us.

My proposal to fix this is to rename the _email_ worker to _prio_ worker and move the publishing of events to the _prio_ worker. If this approach is approved I can code up a PR for this.

**Comment by iglocska** [comment]
This sounds like a great idea indeed. Moving the publishes to a separate
worker is a great idea!

**Comment by iglocska** [comment]
Just one more idea: Maybe instead of renaming the email worker, creating an additional worker with its own queue for some prio tasks would make more sense. I think that in our case for example the e-mail worker is the one that tends to end up with the lognest queues during a sync.

**Comment by RichieB2B** [comment]
Ok, I will add another worker. After this is implemented, would adding a prio flag to the servers make sense? So most servers use the default worker but some can use the prio worker. Or is doing 2 pulls in parallel a no-go?

**Comment by rotanid** [comment]
this got fixed and merged by #1303 , please open a new issue if you want to discuss/suggest something else or similar.
